README.md

```diff
@@ -1,39 +1,39 @@
 # Homebrew Pocket-Core
 
-Repository dedicated to host the formula to install the Pocket Core binary using brew.
+Repository dedicated to host the formula to install the Pocket-Core binary using Homebrew.
 
 ## Getting Started
 
-These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. Remember to always read the documentation available on [Pocket Documentation](https://docs.pokt.network/docs).
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. Remember to always read the documentation available in the [Pocket Documentation](https://docs.pokt.network/docs).
 
 ### Prerequisites
 
-You should have brew install.
+You need to have the package manager Homebrew installed.
 
-To install Homebrew on Mac:
+To install Homebrew on Mac, run the following command in a terminal:
 
-```bash
-/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```zsh
+sudo /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
Why does this need to be sudo?

Reply:
on most macs this is the output without sudo, 
```
% /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)" 
==> Checking for `sudo` access (which may request your password)...
Password:
```

So i suppose it's not needed in the command but you need the password whether or not you use sudo in the command.
```suggestion
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```
I can remove it if you desire.